src/benchmark_environments/mujoco.py

```diff
@@ -0,0 +1,22 @@
+import gym
+from gym.envs.mujoco import half_cheetah_v3
+
+class HalfCheetahEnv(half_cheetah_v3.HalfCheetahEnv):
+    def __init__(self, *args, time_horizon=200, **kwargs):
+        self._time_horizon = time_horizon
+        self._step_count = 0
+        kwargs['exclude_current_positions_from_observation'] = False
+        super().__init__(*args, **kwargs)
+
+    @property
+    def done(self):
+        return self._step_count >= self._time_horizon
```

Comment:
It might be simpler to use the Gym `TimeLimitWrapper` (you can specify `max_episode_steps` in `register` I think as a shortcut), and then just always return `done` as `False` in `step`.